railib/api.py

```diff
@@ -413,7 +413,7 @@ def data(self):
             "dbname": self.database,
             "nowait_durable": self.nowait_durable,
             "readonly": self.readonly,
-            # "sync_mode": "async"
```

Comment:
just repeating from the meeting: I think we should leave this unset and let the engine decide by default. The time threshold is implemented in rai-server via this channel here: https://github.com/RelationalAI/raicode/blob/7a0919e559d020185420c90a92f7a9f0615c33b1/src/Server/Server.jl#L1143